apps/cnspec/cmd/scan.go

```diff
@@ -374,8 +376,33 @@ This example connects to Microsoft 365 using the PKCS #12 formatted certificate:
 		}
 
 		logger.DebugDumpJSON("report", report)
+		// TODO: ONLY FOR TESTING PURPOSES REMOVE ME
+		if fileName := os.Getenv("DEBUG_SAVE_REPORT_COLLECTION_FILE"); fileName != "" {
+			reportBytes, err := proto.Marshal(report)
+			if err != nil {
+				log.Fatal().Err(err).Msg("failed to marshal report")
+			}
+			filePath := "/tmp/" + fileName
+			if err := os.WriteFile(filePath, reportBytes, 0o660); err != nil {
+				log.Fatal().Err(err).Msg("failed to write out report collection")
+			}
+			log.Debug().Str("filePath", filePath).Msg("saved report collection")
+		}
+		// ^^
 		printReports(report, conf, cmd)
 
+		// if no config set, and wasn't explicitly incognito, offer to upload
+		// report for web UI viewing
+		opts, optsErr := cnspec_config.ReadConfig()
+		if optsErr != nil {
+			log.Fatal().Err(optsErr).Msg("could not load configuration")
+		}
+		if opts.GetServiceCredential() == nil && !viper.GetBool("incognito") {
```

Comment:
we also need to check for tty since it would not be great to ask in CI/CD:

```
isTTY := isatty.IsTerminal(os.Stdout.Fd())
```

Reply:
Yeah, that would be unfortunate for CI/CD. Thanks